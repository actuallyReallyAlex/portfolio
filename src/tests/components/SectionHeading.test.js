import React from 'react'
import SectionHeading from '../../components/SectionHeading'
import { shallow } from 'enzyme'

const defaultProps = {
  heading: 'Heading',
  subHeading: 'Sub Heading'
}

describe('<SectionHeading />', () => {
  afterEach(() => {
    // defaultProps.handler.mockClear()
  })

  test('Should render the <SectionHeading /> component.', () => {
    const component = shallow(<SectionHeading {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
