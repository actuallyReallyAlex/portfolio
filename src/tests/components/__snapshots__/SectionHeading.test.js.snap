// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionHeading /> Should render the <SectionHeading /> component. 1`] = `ShallowWrapper {}`;
